empty-plugin/src/main/resources/archetype-resources/pom.xml

```diff
@@ -9,27 +9,45 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>${artifactId}</artifactId>
-    <version>${version}</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.164.1</jenkins.version>
+        <revision>${version}</revision>
```

Comment:
Is this actually right? IIUC this will be `1.0-SNAPSHOT` where we want just `1.0`.